packages/apps/frontera/src/routes/organizations/src/components/FinderTable/FinderTable.tsx

```diff
@@ -194,7 +194,7 @@ export const FinderTable = observer(() => {
       onFocusedRowChange={setFocusIndex}
       onSelectedIndexChange={setSelectedIndex}
       onSelectionChange={handleSelectionChange}
-      enableKeyboardShortcuts={!isEditing || !isFiltering}
```

Comment:
this is not right. I also want to block keyboard events if I'm only searching. The correct fix would be to do store.ui.setIsEdittingFilterTable(true) when you focus the editable input (whicever might be) and set it bag to false when you blur it.

Reply:
that is already done and the condition is correct
if I'm editing then is edit is true so !isEdit is false so I go to or condition and check !isFiltering and cause I'm not filtering at this moment isFiltering is false so !isFiltering is true -> that results in events not being blocked

Reply:
in other words I want keyboard shortcuts to be enabled only when user is not editing and not searching in given moment

Reply:
ok